tensorboard/webapp/runs/views/runs_table/runs_table_container.ts

```diff
@@ -353,9 +376,51 @@ export class RunsTableContainer implements OnInit {
       );
     }
 
+    /**
+     * For consumers who show checkboxes, notify users that new runs may not be
+     * selected by default. Avoid showing it more than once per route, since it
+     * would be annoying to see the alert on every auto-reload (assuming a new
+     * run per reload).
+     *
+     * Warning: this pattern is not recommended in general. Dispatching
+     * `alertReported` would be better handled in a Ngrx Reducer in response
+     * to `fetchRunsSucceeded` or via the declared alert registrations using
+     * `alertFromAction`. Unfortunately, those currently have no way of knowing
+     * whether a run table is actually shown (with checkboxes), so we make a
```

Comment:
Just so we are 100% aligned; since the view is a representation of the state in the store, you _can_ reason with the state in the store to know. However, it is quite cumbersome and, now, you have to ingrain presentation logic into the store which is very unpleasant.